src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/MotionIndex.cs

```diff
@@ -42,14 +42,14 @@ namespace Tizen.NUI.Scene3D
     /// - ModelNodeId (null)       , BlendShapeIndex (string key) : Target is all ModelNode's BlendShape
```

Comment:
line 29 - "whole kind of" -> "all kinds of" 
line 30 - I would close the `<summary>` before or after that line, the summary is too long, rest should go into `<remarks>`
line 36 - the indenting will not affect generated HTML, even line breaks will not be preserved. You could use HTML `<table>` if necessary. Docfx has some support for markdown too (but I would need to experiment with that if you want me to say more about it).

Reply:
Fix done. I use <table> tag for it now.